com/yadro/HWManager/StorageManager.interface.yaml

```diff
@@ -0,0 +1,62 @@
+description: >
+    The interface to control Yadro storage subsystem.
+
+methods:
+    - name: FindDrive
+      description: >
+          Return port name where the rive insrted.
+      parameters:
+          - name: DriveSN
+            type: string
+            description: >
+                Serial number of the drive, to looking for
+      returns:
+          - name: Result
+            type: string
+            description: >
+                Port name
+      errors:
+          - xyz.openbmc_project.Common.Error.InvalidArgument
+          - xyz.openbmc_project.Common.Error.InternalFailure
+          - xyz.openbmc_project.Common.Error.ResourceNotFound
+
+    - name: SetDriveLocationLED
+      description: >
+          Turn On/Off drive location LED.
+      parameters:
+          - name: DriveSN
+            type: string
+            description: >
+                Serial number of the drive, corresponding to required LED
+          - name: Assert
+            type: boolean
+            description: >
+                Requested LED state
+      errors:
+          - xyz.openbmc_project.Common.Error.InvalidArgument
+          - xyz.openbmc_project.Common.Error.InternalFailure
+          - xyz.openbmc_project.Common.Error.ResourceNotFound
+
+    - name: GetDriveLocationLED
+      description: >
+          Request drive location LED status.
+      parameters:
+          - name: DriveSN
+            type: string
+            description: >
+                Serial number of the drive, corresponding to required LED
+      returns:
+          - name: Result
+            type: boolean
+            description: >
+                True if LED asserted, False otherwise
```

Comment:
"True if the LED is on, ..."

Reply:
But in is not literally "ON" - it is blinking. Physically, there is one two-color LED, something like this:
* Green on - drive present, IDLE
* Green blinking - drive activity
* Red on - fault
* Red blinking - Location mode

That's why I don't like to use terms ON/OFF here....

Reply:
Then, I guess, it's "False if the LED is off, True otherwise" :)

Reply:
For the history: I checked google and it appears that they "assert LEDs" here and there in every second datasheet that has anything to do with LEDs. I still think that it's an appropriate use of the verb, but ok. Let it be.